samples/sensor/mcp9600/boards/nrf52840dongle_nrf52840.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020, SER Consulting LLC  / Steven Riedl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
+	mcp9600@60 {
+		compatible = "microchip,mcp9600";
```

Comment:
Need the binding `dts/bindings/sensor/microchip,mcp9600`.

Reply:
need to figure this one out

Reply:
Never knew about that... Found and added